docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,358 @@
+  <figcaption>Accounts Page</figcaption>
+</figure>
+
+In addition, we’ve integrated telemetry into our system to continuously enhance your experience with our products. Using this same infrastructure, you'll soon be able to manage your license without changing your key—thanks to server-side license validation.
+
+Telemetry is released as an opt-in feature. We recommend enabling it by adding the following code to your project, close to your license validation:
+
+```js
+import { muiXTelemetrySettings } from '@mui/x-license';
+
+muiXTelemetrySettings.enableTelemetry();
+```
+
+You can find more details in our [telemetry guide](/x/guides/telemetry/).
+
+## Priority Support – A New Support Service
+
+A few months ago, we launched Priority Support with a few customers, and now we’re excited to invite more users to take advantage of this service.
+
+Priority Support offers fast, dedicated assistance with guaranteed response times, ensuring that technical issues on both MUI X and MUI Core are addressed swiftly and efficiently.
+
+For full details on the benefits and service levels of Priority Support, please see our [Support SLA](https://mui.com/legal/technical-support-sla/).
+
+If you’re interested in upgrading to Priority Support, don’t hesitate to [contact our sales team](mailto:sales@mui.com) for more information or purchase it directly through your account page.
+
+## Support for Material UI
+
+When planning this major release, our goal was to support the last two major versions of Material UI. However, we faced a tough choice: continue supporting older versions (v5 and v6) or fully embrace the innovations of **Material UI v7**. With native ESM support, improved package management, and seamless integration with modern bundlers like Vite and webpack, v7 represents the future of Material UI. Supporting both wasn’t feasible, so we chose to move forward.
+
+As a result, **MUI X v8 now supports Material UI v7 natively**, ensuring your applications benefit from the latest improvements. We understand that change can be challenging, but migrating to v7 has been designed to be as smooth as possible—with comprehensive migration guides and GitHub support to assist you every step of the way.
+
+## Migration Guide
+
+We have built a migration guide for Material UI and one for each of the advanced components. They detail every breaking change from previous versions with the recommended approach to fix them, ensuring that you can transition smoothly without the least amount of hassle. We care about an smooth migration, and that's been a priority for us when planning our new major versions.
+
+Please follow the instructions in our [migration guide](/x/migration/).
+
+## Long-Term Support
+
+We're now moving **MUI X v7 to Long-Term support**, while support for **MUI X v6 is discountinued**. In addition to critical bug fixes and security updates, v7 will continue to receive some bug fixes as a safe net for your your transition for Material UI v7.
```

Comment:
> as a safe net for your your transition for Material UI v7.

Did you mean to mention Material UI v7 here? Shouldn't it be "MUI X v8" instead?